docs/_docs/Starters Academy (LDSSA)/06-Remote-Experience.md

```diff
@@ -49,91 +40,71 @@ then just work away for the rest of the time.
+If you want to enter a side room, you can leave the auditorium 
+and go into the side room. Easy-peasy.
 
 ### Slido
 
 We have all been in that work meeting or class where that one person forgets to mute themselves, 
 making a huge mess of the meeting. Managing so many people in a meeting and making sure everyone 
 gets a turn is a big challenge. To avoid confusion and give everyone a chance to clarify the 
-presentation content or just ask more about a topic, the Q&A this year will make use of slido 
+presentation content or just ask more about a topic, the Q&A will make use of slido 
 (https://www.sli.do/).
 
 This is a platform where you can write your own questions or vote for other people’s questions 
-to be answered. Without any need for an account, you can just access the website and introduce 
-a slide code, which will be given both in the beginning of each presentation and at the time of 
-Q&A. The instructor will then go through the most voted questions until their out of time. 
-
-But even if your question doesn’t get answered live, it can still get answered. First, you can 
-take advantage of the rooms to discuss those questions left and answered. But additionally, at 
-the end of the bootcamp all instructors will try to compile a list of questions and answers that 
-will then be shared.
+to be answered. Without any need for an account, you can access the website and introduce 
+a slido code, which will be shared in all virtual meetings.
 
 ### Slack
 
-This is how you have written conversations and ask questions that don't require an immediate 
-response. The majority of your experience in the academy will be on slack. Please note that 
-there are rooms created for each of the SLUs, BLUs, and hackathons. Be sure to post your question 
+Slack in our communication tool during all the Academy. All important announcements will be made via Slack.
+This is where you can have written conversations and ask questions that don't require an immediate 
+response. Please note that there are channels created for each of the SLUs, BLUs, and hackathons. Be sure to post your question 
 in the right place!
 
 Note that we have not paid for a full slack subscription. This is on purpose. Slack is not a place 
-that should generate a source of truth for anything at all. Slack is for information that is only 
-meant to last for a few days. Anything else needs to go into something official like a github issue 
-or the wiki if you want it preserved. This is because finding information on slack that is more 
-than a few days old is a nightmare.
-
-The most important thing you can do for the bootcamp days is to make sure that you're paying 
-attention to the announcements channel.
-
+that should generate a source of truth for anything at all. Slack messages disappear after 90 days.
+Anything else needs to go into something official like a github issue 
+or the wiki if you want it preserved.
```

Comment:
Are we encouraging the students to submit changes to the wiki here? Does it make sense?

Reply:
It doesn't :) I removed it.